packages/orchestrator-ui-components/src/components/WFOForms/formFields/index.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019-2023 SURF.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+//@ts-nocheck
+// export { default as AutoFields } from './AutoFields';
```

Comment:
Afaik remove these commented imports and add them back when they become relevant again